JobBars/Data/Levels.Action.cs

```diff
@@ -0,0 +1,246 @@
+using System.Collections.Generic;
```

Comment:
this is going to be really annoying to maintain. A better way is to just get these values from the Excel sheet like https://github.com/0ceal0t/JobBars/blob/main/JobBars/Jobs/DRK.cs#L65

`Action` has a property like `ClassJobLevel` which has this information

Reply:
Replaced `Levels.Action.cs` with an implementation in `AtkHelper` using data from Lumina as suggested